diff --git a/node_modules/react/cjs/react.development.js b/node_modules/react/cjs/react.development.js
index 234f4b0..7c48c87 100644
--- a/node_modules/react/cjs/react.development.js
+++ b/node_modules/react/cjs/react.development.js
@@ -1326,7 +1326,7 @@
     exports.useTransition = function () {
       return resolveDispatcher().useTransition();
     };
-    exports.version = "19.2.0-experimental-c129c242-20250505";
+    exports.version = "19.2.0-experimental-0ca8420f-20250504";
     "undefined" !== typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ &&
       "function" ===
         typeof __REACT_DEVTOOLS_GLOBAL_HOOK__.registerInternalModuleStop &&
diff --git a/node_modules/react/cjs/react.production.js b/node_modules/react/cjs/react.production.js
index e98c79b..c22b875 100644
--- a/node_modules/react/cjs/react.production.js
+++ b/node_modules/react/cjs/react.production.js
@@ -603,4 +603,4 @@ exports.useSyncExternalStore = function (
 exports.useTransition = function () {
   return ReactSharedInternals.H.useTransition();
 };
-exports.version = "19.2.0-experimental-c129c242-20250505";
+exports.version = "19.2.0-experimental-0ca8420f-20250504";
diff --git a/node_modules/react/cjs/react.react-server.development.js b/node_modules/react/cjs/react.react-server.development.js
index c1a91c9..fa37015 100644
--- a/node_modules/react/cjs/react.react-server.development.js
+++ b/node_modules/react/cjs/react.react-server.development.js
@@ -992,5 +992,5 @@
     exports.useMemo = function (create, deps) {
       return resolveDispatcher().useMemo(create, deps);
     };
-    exports.version = "19.2.0-experimental-c129c242-20250505";
+    exports.version = "19.2.0-experimental-0ca8420f-20250504";
   })();
diff --git a/node_modules/react/cjs/react.react-server.production.js b/node_modules/react/cjs/react.react-server.production.js
index ae987bf..2dac15e 100644
--- a/node_modules/react/cjs/react.react-server.production.js
+++ b/node_modules/react/cjs/react.react-server.production.js
@@ -569,4 +569,4 @@ exports.useId = function () {
 exports.useMemo = function (create, deps) {
   return ReactSharedInternals.H.useMemo(create, deps);
 };
-exports.version = "19.2.0-experimental-c129c242-20250505";
+exports.version = "19.2.0-experimental-0ca8420f-20250504";
diff --git a/node_modules/react/package.json b/node_modules/react/package.json
index d9ce355..ab201cc 100644
--- a/node_modules/react/package.json
+++ b/node_modules/react/package.json
@@ -4,7 +4,7 @@
   "keywords": [
     "react"
   ],
-  "version": "0.0.0-experimental-c129c242-20250505",
+  "version": "0.0.0-experimental-0ca8420f-20250504",
   "homepage": "https://react.dev/",
   "bugs": "https://github.com/facebook/react/issues",
   "license": "MIT",
